src/services/codefixes/fixUnusedIdentifier.ts

```diff
@@ -100,7 +98,7 @@ namespace ts.codefix {
                     // or "'import {a, b as ns} from './file'"

```

Comment:
actually this comment is wrong. I'm removing it. The correct cases are already articulated in inline comments just below.